public/app/features/library-panels/components/LibraryPanelsView/LibraryPanelsView.tsx

```diff
@@ -33,6 +34,7 @@ export const LibraryPanelsView = ({
   showSecondaryActions,
   currentPanelId: currentPanel,
   perPage: propsPerPage = 40,
+  isWidget,
```

Comment:
Maybe here we could add another `filter` function? The same proposal as before? 
```jsx
const isWidget = (plugin) => plugin.meta.skipDataQuery;

<LibraryPanelsView filter={isWidget} />
```

Reply:
Tackling this suggestion after the POC is tested :)